src/components/button/button.scss

```diff
@@ -19,9 +19,6 @@
 }
```

Comment:
Is it intentional that the text colour becomes black when a non-primary button is hovered?

<img width="1440" alt="image" src="https://user-images.githubusercontent.com/154725/211854419-7e2c4416-1f4c-404b-a019-fd929a63adf6.png">

Reply:
caused by the previous problem

Reply:
I assume "the previous problem" refers to the problem with the button colours? I thought that was the case too, but that problem has been fixed now, while this issue remains.

But perhaps this wasn't fixed because I opted to only import those colour variables in `button.scss`, not remove the whole commit?

Regardless, this doesn't seem to be a problem on buttons where the colours have been set explicitly by the consumer, and with the default colours this certainly isn't a show-stopper, in my opinion.